index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id = "main"> 
+      <h1 id = "title">Dr.Norman Borlaug</h1>
+      <figure id="img-div"> 
+        <img id="image" src = "https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"> 
+        <figcaption id = "img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figurecation>
+      </figure>
```

Comment:
please add a 'white' background to the figure section.

Reply:
- [ ] Please refer to my detailed review on line `11` above on how to add a `why for suggestions made 👍